provisioner.go

```diff
@@ -185,8 +185,13 @@ func (p *defaultProvisioner) provisionElectron(ctx context.Context, d Dispatcher
 
 // provisionPackage provisions a package
 func (p *defaultProvisioner) provisionPackage(ctx context.Context, d Dispatcher, paths Paths, s *ProvisionStatusPackage, m mover, name, version, pathUnzipSrc, pathDirectory string, finish func() error) (err error) {
+	_, err = os.Stat(pathDirectory)
```

Comment:
@asticode because we would like to distribute our desktop application to user with the electron and astilectron. Because user maybe use our application on an internal network which cannot visit github. And of course, from here download github is some slow and we don't want user to wait about 1 minute to download the electron.